yb-voyager/cmd/assessMigrationCommandBulk.go

```diff
@@ -0,0 +1,311 @@
+	BoolVar(assessMigrationBulkCmd.Flags(), &continueOnError, "continue-on-error", true, "If true, it will print the error message on console and continue to next schema’s assessment")
+	assessMigrationBulkCmd.Flags().Bool("ignore-exists", true, "If true, skip assessment if for a schema it's already done")
+	assessMigrationBulkCmd.Flags().StringVar(&bulkAssessmentDir, "bulk-assessment-dir", "", "Top-level directory storing the export-dir of each schema (default: pwd)")
+	BoolVar(assessMigrationBulkCmd.Flags(), &startClean, "start-clean", false, "Cleans up all the export-dirs in bulk assessment directory to start everything from scratch")
+
+	// marking mandatory flags
+	assessMigrationBulkCmd.MarkFlagRequired("fleet-config-file")
+	assessMigrationBulkCmd.MarkFlagRequired("bulk-assessment-dir")
+}
+
+func assessMigrationBulk() {
+	if startClean {
+		proceed := utils.AskPrompt(
+			"CAUTION: Using --start-clean will delete all progress in each export directory present inside the bulk-assessment-dir. " +
+				"Do you want to proceed")
+		if !proceed {
+			return
+		}
+
+		// cleaning all export-dir present inside bulk-assessment-dir
+		matches, err := filepath.Glob(fmt.Sprintf("%s/*-export-dir", bulkAssessmentDir))
+		if err != nil {
+			utils.ErrExit("error while cleaning up export directories: %s", err)
+		}
+		for _, match := range matches {
+			utils.CleanDir(match)
+		}
+
+		err = os.RemoveAll(filepath.Join(bulkAssessmentDir, "bulkAssessmentReport.html"))
+		if err != nil {
+			utils.ErrExit("failed to remove bulk assessment report: %s", err)
+		}
+		utils.CleanDir(filepath.Join(bulkAssessmentDir, "logs"))
+	}
+
+	dbConfigs, err := parseFleetConfigFile(fleetConfigPath)
+	if err != nil {
+		utils.ErrExit("failed to parse fleet config file: %v", err)
+	}
+
+	for _, dbConfig := range dbConfigs {
+		utils.PrintAndLog("\nAssessing '%s' schema\n", dbConfig.Schema)
+		err = executeAssessment(dbConfig)
+		if err != nil {
+			log.Errorf("failed to assess migration for schema %s: %v", dbConfig.Schema, err)
+			fmt.Printf("failed to assess migration for schema %s: %v\n", dbConfig.Schema, err)
+			if !continueOnError {
+				break
+			}
+		}
+	}
+
+	err = generateBulkAssessmentReport(dbConfigs)
+	if err != nil {
+		utils.ErrExit("failed to generate bulk assessment report: %s", err)
+	}
+}
+
+func executeAssessment(dbConfig AssessMigrationDBConfig) error {
+	log.Infof("executing assessment for schema %q", dbConfig.Schema)
+	exportDirPath := getAssessmentExportDirPath(dbConfig)
+	cmdArgs := buildCommandArguments(dbConfig, exportDirPath)
+	if err := os.MkdirAll(exportDirPath, 0755); err != nil {
+		return fmt.Errorf("creating export-directory %q for schema %q: %w", exportDirPath, dbConfig.Schema, err)
+	}
+
+	execCmd := exec.Command(os.Args[0], cmdArgs...)
+	execCmd.Stdout = os.Stdout
+	execCmd.Stderr = os.Stderr
+	execCmd.Stdin = os.Stdin
+	log.Infof("executing the cmd: %s", execCmd.String())
```

Comment:
Please take care of the password param in logging as I see we are passing it with flag to assess-migration.